### ISSUE TITLE
extension is removed from chrome store

### ISSUE THREAD

**Issue by elievim** [opus1269/screensaver#36]
this makes me very sad, whats the reason?

**Comment by SilverMonkey1** [comment]
> this makes me very sad, whats the reason?

I loved this extension.

**Comment by jadamnoah** [comment]
I agree - this is my favorite extension! -- I run it on all of my computers and lets me see many happy pictures throughout the day.

**Comment by ndmax** [comment]
I'm not certain, but I believe the author (Michael A Updike) may be deceased:

[Stack Overflow Bio](https://stackoverflow.com/users/4468645/michael-updike)
[Obituary](https://www.legacy.com/us/obituaries/kendallcountynow/name/michael-updike-obituary?id=9278882)

**Comment by tobinbc** [comment]
The software we write may outlive us and keep bringing joy ❤️
